app/views/publish/courses/study_mode/edit.html.erb

```diff
@@ -2,13 +2,22 @@
 <% content_for :page_title, title_with_error_prefix("#{page_title} – #{course.name_and_code}", @course_study_mode_form.errors.any?) %>
 
 <% content_for :before_content do %>
-  <%= govuk_back_link_to(
-        details_publish_provider_recruitment_cycle_course_path(
-          course.provider_code,
-          course.recruitment_cycle_year,
-          course.course_code
-        )
-      ) %>
+  <% if params[:previous_tda_course] == 'true' %>
+    <%= govuk_back_link_to(funding_type_publish_provider_recruitment_cycle_course_path(
+            course.provider_code,
+            course.recruitment_cycle_year,
+            course.course_code,
+            previous_tda_course: true
+          )) %>
+  <% else %>
+    <%= govuk_back_link_to(
+          details_publish_provider_recruitment_cycle_course_path(
+            course.provider_code,
+            course.recruitment_cycle_year,
+            course.course_code
+          )
+        ) %>
+  <% end %>
```

Comment:
Though the logic is less complex here (so I wouldn't necessarily block it over this), it's not a pattern we want to expand. Generally something to be cautious about. If your erbs are full of logic, are they actually just different views?

Reply:
Done here too: https://github.com/DFE-Digital/publish-teacher-training/pull/4284/commits/3acddbbd14034e43ec8b9a690907e634eefd2e3d